jolt-core/src/jolt/instruction/test.rs

```diff
@@ -28,3 +28,40 @@ macro_rules! jolt_instruction_test {
         assert_eq!(actual, expected, "{:?}", $instr);
     };
 }
+
+use ark_ff::PrimeField;
+use crate::jolt::instruction::JoltInstruction;
+use rand_core::SeedableRng;
+
+const TEST_C: usize = 4;
+const TEST_M: usize = 1 << 16;
+
+/// Tests `num_iters` random instructions for their parity between the `lookup_entry()`
+/// and `lookup_entry_u64` functions. Needs `concrete_instruction: I` to call `.random(&self)`.
+pub fn lookup_entry_u64_parity_random<F, I>(num_iters: usize, concrete_instruction: I) 
+where
+    F: PrimeField,
+    I: JoltInstruction
+{
+    let mut rng = rand::rngs::StdRng::seed_from_u64(1234567890);
+    let mut instructions: Vec<I> = Vec::new();
+    for _ in 0..num_iters {
+        let instruction = concrete_instruction.random(&mut rng);
+        instructions.push(instruction);
+    }
+    lookup_entry_u64_parity::<F, I>(instructions);
+}
+
+/// Tests `instructions` for their parity between the `lookup_entry()`
+/// and `lookup_entry_u64` functions.
+pub fn lookup_entry_u64_parity<F, I>(instructions: Vec<I>) 
+where
+    F: PrimeField,
+    I: JoltInstruction 
+{
+    for (index, instruction) in instructions.into_iter().enumerate() {
+        let fr = instruction.lookup_entry::<F>(TEST_C, TEST_M);
+        let u = instruction.lookup_entry_u64();
+        assert_eq!(fr, F::from(u), "[{index}] Entries did not match for {instruction:?}. Left: lookup_entry(), Right: lookup_entry_u64()");
+    }
+}
```

Comment:
let's delete these; we really only need `lookup_entry_u64` and the `jolt_instruction_test` macro

Reply:
done